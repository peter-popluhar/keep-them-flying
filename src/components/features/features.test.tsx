import { render, screen } from "@testing-library/react";
import Features from "./features";

test("renders Features", () => {
  const { container } = render(<Features />);
  const text = screen.getByText(
    /Solutions for travelers during the COVID-19 pandemic/i
  );
  expect(text).toBeInTheDocument();
  expect(container).toMatchSnapshot();
});
