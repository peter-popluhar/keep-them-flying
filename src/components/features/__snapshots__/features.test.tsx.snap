// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Features 1`] = `
<div>
  <div
    class="features"
  >
    <h2>
      Solutions for travelers during the COVID-19 pandemic
    </h2>
    <ul
      class="list"
    >
      <li>
        <img
          alt="Refunds & cancellations"
          src="https://www.kiwi.com/images/hero/icon-refunds.svg"
        />
        <h3>
          Refunds & cancellations
        </h3>
        <p>
          Learn more about our numerous refund options like carrier self-service and instant refunds.
        </p>
        <button
          class="ant-btn ant-btn-default"
          type="button"
        >
          <span>
            Find out more
          </span>
        </button>
      </li>
      <li>
        <img
          alt="Request a refund"
          src="https://www.kiwi.com/images/hero/icon-self-service.svg"
        />
        <h3>
          Request a refund
        </h3>
        <p>
          The fastest and easiest way to cancel your booking is online via our refunds & cancellations form.
        </p>
        <button
          class="ant-btn ant-btn-default"
          type="button"
        >
          <span>
            Manage my booking
          </span>
        </button>
      </li>
      <li>
        <img
          alt="Coronavirus information"
          src="https://www.kiwi.com/images/hero/icon-information.svg"
        />
        <h3>
          Coronavirus information
        </h3>
        <p>
          Read about how the new travel restrictions might affect your trip.
        </p>
        <button
          class="ant-btn ant-btn-default"
          type="button"
        >
          <span>
            COVID-19 information
          </span>
        </button>
      </li>
    </ul>
  </div>
</div>
`;
